internal/wmdocker/client_benchmark_test.go

```diff
@@ -0,0 +1,31 @@
+//+build !skipcontainertests
+
+package wmdocker
+
+// func BenchmarkExecInContainer(b *testing.B) {
```

Comment:
why did this get commented out? i'm pretty sure we still want this to compile

Reply:
lol whoops

Reply:
oh right, it's cuz i need to make everything on the DockerBuildFixture public and it was a pita. but it's fine.

Reply:
ugh nvm, it's a giiiant pain and leads to import cycles. imo the `DockerFixture` should maybe live in `docker/testutils.go` but there's a lotttt of stuff to disentangle. however i assume it would be totally gross to put `client_benchmark_test.go` in the `build` package to avoid import cycles/having to split out the docker fixture?

Reply:
oh that's fine just put it in the build package for now